virtio-queue/src/queue/verification.rs

```diff
@@ -269,3 +504,205 @@ fn verify_spec_2_7_7_2() {
         // So we do not care
     }
 }
+
+#[kani::proof]
+#[kani::unwind(0)]
+fn verify_add_used() {
+    let ProofContext { mut queue, memory } = kani::any();
+    let used_idx = queue.next_used;
+    kani::assume(used_idx.0 < queue.size());
+    let used_desc_table_index = kani::any();
+    if queue
+        .add_used(&memory, used_desc_table_index, kani::any())
+        .is_ok()
+    {
+        assert_eq!(queue.next_used, used_idx + Wrapping(1));
+    } else {
+        assert_eq!(queue.next_used, used_idx);
+
+        // Ideally, here we would want to actually read the relevant values from memory and
```

Comment:
I think this comment is true in firecraker but is it still valid in the rust-vmm implementation?